Libraries/src/Amazon.Lambda.PowerShellHost/Amazon.Lambda.PowerShellHost.csproj

```diff
@@ -3,20 +3,20 @@
   <Import Project="..\..\..\buildtools\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Is net6.0 target needed?

Reply:
Technically somebody could stick with the old major version of the module and have a "PowerShell Lambda .NET project" and update to this new major version of Amazon.Lambda.PowerShellHost. An unlikely scenario but since .NET 6 is still in support I'm not ready to rule it out.